src/swarm/neo/node/IRequest.d

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+
+    Interface for a node request handler object.
+
+    Note that the `ConnectionHandler` (which uses this interface) requires
+    implementing classes to have neither constructors nor destructors. The
+    reason for this is documented in `ConnectionHandler.emplace`.
+
+    copyright: Copyright (c) 2018 dunnhumby Germany GmbH. All rights reserved
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module swarm.neo.node.IRequest;
```

Comment:
This rename is a breaking change and should have a release note (for the sanity of library users).

Reply:
I added a release note for this.